HPC/README.md

```diff
@@ -80,7 +80,7 @@ tar xf arm-compiler-for-linux_23.04_Ubuntu-20.04_aarch64.tar
 
 # load the module to use Arm Compiler for Linux (ACfL)
 module use /shared/arm/modulefiles
-module load acfl/23.04
+module load acfl
```

Comment:
I hope you validated this works without explicit version. In such a case, if there are multiple versions installed, which one would be picked up?

Reply:
Yes, I have verified. The default armpl installed on parallel cluster is 21.0.0. I have manually installed latest version to /shared/arm.  By doing `module use /shared/arm/modulefiles`, the newer library is put ahead of the default module. 

```
$ echo $MODULEPATH
/etc/environment-modules/modules:/usr/share/modules/versions:/usr/share/modules/$MODULE_VERSION/modulefiles:/usr/share/modules/modulefiles
$ module use /shared/arm/modulefiles/
$ echo $MODULEPATH
/shared/arm/modulefiles:/etc/environment-modules/modules:/usr/share/modules/versions:/usr/share/modules/$MODULE_VERSION/modulefiles:/usr/share/modules/modulefiles
```